docs/learn/tutorials/versioned/hello-samza-high-level-yarn.md

```diff
@@ -63,7 +62,7 @@ Then, you can continue w/ the following command in hello-samza project:
 {% highlight bash %}
 mvn clean package
 mkdir -p deploy/samza
-tar -xvf ./target/hello-samza-1.0.1-SNAPSHOT-dist.tar.gz -C deploy/samza
+tar -xvf ./target/hello-samza-1.1.0-dist.tar.gz -C deploy/samza
```

Comment:
Previously we used to have `SNAPSHOT` string in the samza-1.0-dist tar-ball. Can you please share the rationale for removing it? Is it not necessary anymore?

Reply:
The rationale is that now that this is the latest release branch we no longer need to have SNAPSHOT. The previous has SNAPSHOT since it this release branch is branched off of maste, which contains the latest changes non in a version (therefore we needed SNAPSHOT). This process is documented here: https://github.com/apache/samza/blob/master/docs/README.md#release-new-version-website-checklist

Reply:
Thanks for the clarification.